src/words/environment/WordsObject.java

```diff
@@ -95,6 +95,12 @@ public Property getProperty(String propertyName) {
 			return new Property(cell.y);
 		else if (propertyName.equals("column"))
 			return new Property(cell.x);
+		
+		// Special handling for Object name "name" and Class name "class"
+		if (propertyName.equals("name"))
```

Comment:
As Alex suggested, this should just be a continuation of the above if (as another else if) rather than a new if.